hazelcast-client/src/main/java/com/hazelcast/client/spi/impl/HazelcastClientNotActiveException.java

```diff
@@ -0,0 +1,16 @@
+package com.hazelcast.client.spi.impl;
+
+/**
+ * Thrown when Hazelcast client is not active during an invocation.
+ */
+public class HazelcastClientNotActiveException extends IllegalStateException {
```

Comment:
Yes, it will be send to user, but wrapped in HazelcastException. I will change the package.